lightning/src/ln/peer_handler.rs

```diff
@@ -755,6 +755,7 @@ impl<Descriptor: SocketDescriptor, CM: Deref> PeerManager<Descriptor, CM> where
 
 											// Unknown messages:
 											wire::Message::Unknown(msg_type) if msg_type.is_even() => {
+												log_debug!(self, "Received unknown even message of type {}", msg_type);
```

Comment:
I would log this for both even and odd messages, and just only return the error when the message is even.